bindgen/bindgen.bzl

```diff
@@ -168,16 +168,16 @@ rust_bindgen = rule(
             doc = "Flags to pass directly to the clang executable.",
         ),
         "_process_wrapper": attr.label(
-            default = "@io_bazel_rules_rust//util/process_wrapper",
+            default = Label("//util/process_wrapper"),
             executable = True,
             allow_single_file = True,
             cfg = "exec",
         ),
     },
     outputs = {"out": "%{name}.rs"},
     toolchains = [
-        "@io_bazel_rules_rust//bindgen:bindgen_toolchain",
-        "@io_bazel_rules_rust//rust:toolchain",
+        str(Label("//bindgen:bindgen_toolchain")),
```

Comment:
I'm shopping around for some thoughts here and generally more insight on [Label](https://docs.bazel.build/versions/master/skylark/lib/Label.html). I'm not sure if this is an appropriate use of `Label`. Anyone have any experiences like this they can share?

Reply:
This is perfectly acceptable. I remember having a funny discussion with several senior engineer in the Bazel team about this "hack" and having the manager answer: I don't think this is a hack, which just settled our decision to keep this as an idiom, although I left the team 3 years ago so I don't know what is the status now.